peer/chaincode.proto

```diff
@@ -85,3 +87,44 @@ message ChaincodeInvocationSpec {
 message LifecycleEvent {
     string chaincode_name = 1;
 }
+
+// https://github.com/hyperledger/fabric/blob/master/core/common/ccprovider/cdspackage.go#L36
```

Comment:
Please remove the ilnks. They're unstable and unnecessary in the context of the message documentation.

Reply:
done